jsettlers.graphics/src/jsettlers/graphics/reader/translator/SettlerTranslator.java

```diff
@@ -44,9 +44,22 @@ public HeaderType getHeaderType() {
 	}
 
 	@Override
-	public short readUntransparentColor(ByteReader reader)
-			throws IOException {
-		return (short) ((reader.read16() << 1) | 0x01);
+	public short readUntransparentColor(ByteReader reader) throws IOException {
+		int read = reader.read16();
+
+		int R5 = (int)((read & 0xF800) >> 11);
```

Comment:
https://github.com/justangel/settlers-remake/commit/aabfc02ce8cf5684a3e97eb05ac2bbb49f3fce4c right? or other place better to move?